features/resources/programs/box_app.teal

```diff
@@ -0,0 +1,55 @@
+#pragma version 7
```

Comment:
Visual reminder:  Since AVM 7 does _not_ include Boxes, we'll need to bump `pragma version` from 7 to 8 prior to merging and _after_ AVM 7 release.